recipe/meta.yaml

```diff
@@ -28,6 +28,9 @@ test:
   requires:
     - pyqt
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
```

Comment:
Add please `python <3.10`

Reply:
The `python <3.10` has been added to the `requires` section

https://github.com/AnacondaRecipes/qtawesome-feedstock/blob/update_to_1.0.3/recipe/meta.yaml#L34